extensions/src/main/java/org/apache/falcon/extensions/store/ExtensionStore.java

```diff
@@ -250,6 +250,20 @@ public String getExtensionResource(final String resourcePath) throws StoreAccess
         return extesnionList;
     }
 
+    public String deleteExtensionMetadata(final String extensionName) throws StoreAccessException{
+        ExtensionType extensionType = AbstractExtension.isExtensionTrusted(extensionName) ? ExtensionType.TRUSTED
+                : ExtensionType.CUSTOM;
+        if (extensionType.toString().equalsIgnoreCase(ExtensionType.TRUSTED.toString())){
+            throw new StoreAccessException(new Exception(extensionName + " is trusted cannot be deleted."));
```

Comment:
Nit : No need to create new Exception. You can directly create StoreAccessException